packages/react/src/experimental/SelectPanel2/SelectPanel.examples.stories.module.css

```diff
@@ -64,3 +72,12 @@
 .FormControl {
   margin-bottom: var(--base-size-8);
 }
+
+.ButtonContentStartJustify [data-component='buttonContent'] {
+  width: 200px;
```

Comment:
```suggestion
```
already handled with inline styled and shouldn't be inside [data-component='buttonContent']